homeassistant/components/hunterdouglas_powerview/scene.py

```diff
@@ -30,7 +30,7 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def async_setup_platform(hass, config, async_setup_platform, discovery_info=None):
```

Comment:
```suggestion
def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Reply:
Not sure if Black approves :)

Reply:
Thanks! I think I may be context switching a bit too much today.  🤦